test/parameter_test.py

```diff
@@ -261,6 +275,16 @@ def test_enum_param_invalid(self):
     def test_enum_param_missing(self):
         self.assertRaises(ParameterException, lambda: luigi.parameter.EnumParameter())
 
+    def test_list_parameter(self):
+        a = luigi.ListParameter()
+        b_list = [1, 2, 3]
+        self.assertEqual(FooList.a_list, a.parse(a.serialize(b_list)))
```

Comment:
Yes, that is what i'm trying to test. Writing tests is still new to me. Thanks! I'll make that correction.